src/awesome/root.rs

```diff
@@ -84,6 +84,57 @@ fn set_wallpaper<'lua>(_: &'lua Lua, _pattern: *mut cairo_pattern_t) -> rlua::Re
 
 fn tags<'lua>(lua: &'lua Lua, _: ()) -> rlua::Result<Table<'lua>> {
     let table = lua.create_table()?;
-    // TODO FIXME Get tags
+    let activated_tags = lua.named_registry_value::<Table>(super::tag::TAG_LIST)?;
+    for pair in activated_tags.clone().pairs::<Value, Value>() {
```

Comment:
Is this `clone` necessary? It compiles fine without it.

Reply:
Nope, it's not. It only clones the reference anyway. I will remove this.